design-system/materials/internals/definition.yaml

```diff
@@ -74,6 +74,29 @@ choiceGroupsByTheme:
       prefix: font-size
       choices:
         font-size-m: '1rem'
+        font-size-10: '0.75rem'
```

Comment:
I would also update the comment in the header of the file to give another example using variants.
Now:
```
# The names of decisions in this file need to follow this format:
#   <attribute>-for-<component-group>-when-<state>-
```

Proposal:
```
# The names of decisions in this file need to follow this format:
#   <attribute>-for-<component-group>-when-<state>
#   <attribute>-for-<component-group>-as-<state>
```

Also, I think it might be possible to use both combined (nothing we need right now):
```
<attribute>-for-<component-group>-as-<state>-when-<state>
font-size-for-text-as-h2-when-hovered
```

Reply:
Thank you, totally makes sense! Updated  it 😊 
And I agree we might down the line include also the combination.